stable/elasticsearch-exporter/README.md

```diff
@@ -67,7 +67,14 @@ Parameter | Description | Default
 `es.ssl.client.key` | Private key for client auth when connecting to Elasticsearch |
 `web.path` | path under which to expose metrics | `/metrics`
 `serviceMonitor.enabled` | If true, a ServiceMonitor CRD is created for a prometheus operator | `false`
-`serviceMonitor.labels` | Labels for prometheus operator | `{}`
+`serviceMonitor.labels` | ServiceMonitor labels for prometheus operator | `{}`
+`prometheusRule.enabled` | If true, a PrometheusRule CRD is created for a prometheus operator | `false`
+`prometheusRule.prometheusNamespace` | Namespace for prometheusRule | `monitoring`
+`prometheusRule.elasticsearchNodes` | Number of all nodes in Elasticsearch cluster (all roles) | `7`
```

Comment:
Could this be inferred from the values?

Reply:
Sorry. I did not understand the question

Reply:
I mean, could this value be automatically calculated from .Values.client.replicas + .Values.data.replicas + .Values.master.replicas?

Reply:
This chart does not have these variables.

Reply:
Oh, indeed, sorry, I mixed different things in my head.